edc-extensions/provision-additional-headers/src/test/java/org/eclipse/tractusx/edc/provision/additionalheaders/ProvisionAdditionalHeadersExtensionTest.java

```diff
@@ -46,46 +57,69 @@
 @ExtendWith(EdcExtension.class)
```

Comment:
maybe not relevant for this PR, but we could use the `DependencyInjectionExtension` here, so that we don't have to spin up an entire runtime.
In any case the test should be annotated with `@ComponentTest`

Reply:
Yeah i would change that in another PR